src/mmw/js/src/core/utils.js

```diff
@@ -73,6 +73,54 @@ var utils = {
             });
 
         return md5(JSON.stringify(sortedCollection));
+    },
+
+    convertToMetric: function(value, fromUnit) {
```

Comment:
I'm not thrilled with that idea because it allows for the possibility of converting to things that don't make any sense. Ex: lbs -> km 

If this seems a lot more helpful, I'd create more function
convertDistanceToMetric
convertDistanceToImperial
convertMassToMetric
convertMassToImperial

just to avoid a ton of logic to ensure the input and output units are compatible.